example/README.md

```diff
@@ -0,0 +1,52 @@
+# Examples
+
+These examples demonstrate various Ignition Fuel tools features.
+
+ conda install gflags  --channel conda-forge
+
+## Build
+
+After installing Ignition Fuel tools, from source or from binaries, build with:
+
+```
+git clone https://github.com/ignitionrobotics/ign-fuel-tools/
+cd ign-fuel-tools/examples
+mkdir build
+cd build
+cmake ..
+```
+
+### Ubuntu and MacOS
+
+```bash
+make
+```
+
+### Windows
+
+Remember to install the `gflags` dependency:
```

Comment:
We could also say this for the other platforms.

This should also be moved above the build step.